datadog/resource_datadog_monitor.go

```diff
@@ -195,11 +202,15 @@ func resourceDatadogMonitor() *schema.Resource {
 				Type:     schema.TypeBool,
 				Optional: true,
 			},
+			"validate": {
```

Comment:
What happens if the only change is to flip this from default true to false? If there aren't any changes to send to the API, I'm not sure if the endpoint accepts `PUT /monitor/<id> {}`, would be good to verify.

Reply:
You mean if the monitor object is already created, and we change the flag? I'll test it.

Reply:
Good catch, removed it from the diff explicitly.